deepset_cloud_sdk/_utils/ratelimiter.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+import time
+from typing import Any
+
+import aiohttp
+from aiohttp import ClientSession
+
+
+class RateLimiter:
```

Comment:
can we please add unit tests for this class?

Reply:
adddd